upstream-community-operators/myvirtualdirectory/myvirtualdirectory.1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,241 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: myvirtualdirectory.1.0.0
+  annotations:
+    categories: Security
+    description: MyVirtualDirectory Operator to provide LDAP virtual directory services and LDAP proxy services
+    certified: "false"
+    containerImage: docker.io/tremolosecurity/myvd-k8s-operator
+    createdAt: 2019-04-15
+    support: https://github.com/TremoloSecurity/myvd-k8s-operator
```

Comment:
Please place the github repository into the `metadata.annotations.repository` field